nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_hesai/hesai_hw_interface.hpp

```diff
@@ -77,9 +77,12 @@ const uint16_t PANDAR128_E4X_EXTENDED_PACKET_SIZE = 1117;
 
 const uint16_t MTU_SIZE = 1500;
 
-const int PTP_LOG_ANNOUNCE_INTERVAL = 1; // Time interval between Announce messages, in units of log seconds (default: 1)
-const int PTP_SYNC_INTERVAL = 1; //Time interval between Sync messages, in units of log seconds (default: 1)
-const int PTP_LOG_MIN_DELAY_INTERVAL = 0; //Minimum permitted mean time between Delay_Req messages, in units of log seconds (default: 0)
+const int PTP_LOG_ANNOUNCE_INTERVAL =
+  1;  // Time interval between Announce messages, in units of log seconds (default: 1)
+const int PTP_SYNC_INTERVAL =
+  1;  // Time interval between Sync messages, in units of log seconds (default: 1)
+const int PTP_LOG_MIN_DELAY_INTERVAL = 0;  // Minimum permitted mean time between Delay_Req
+                                           // messages, in units of log seconds (default: 0)
```

Comment:
(pedantic) Inline comments causing a line break

Reply:
Addressed in [adf43fe](https://github.com/tier4/nebula/pull/126/commits/adf43fef5af5924675b60e317a6de3b59e7a9f06)